apps/mobile/src/screens/connection.tsx

```diff
@@ -1,14 +1,17 @@
 import { RiseComponents } from '@rise-tools/kit'
-import { ExpoRouterComponents, useExpoRouterActions } from '@rise-tools/kit-expo-router'
-import { FormComponents } from '@rise-tools/kit-forms'
-import { useHapticsActions } from '@rise-tools/kit-haptics'
-import { useLinkingActions } from '@rise-tools/kit-linking'
-import { LucideIconsComponents } from '@rise-tools/kit-lucide-icons'
-import { QRCodeComponents } from '@rise-tools/kit-qrcode'
-import { SVGComponents } from '@rise-tools/kit-svg'
-import { useToastActions } from '@rise-tools/kit-tamagui-toast'
+import {
+  ExpoRouterComponents,
+  FormComponents,
+  LucideIconsComponents,
+  QRCodeComponents,
+  SVGComponents,
+  TamaguiComponents,
+  useExpoRouterActions,
+  useHapticsActions,
+  useLinkingActions,
+  useToastActions,
+} from '@rise-tools/kitchen-sink'
```

Comment:
hmm at first I assumed it would be `import { Components, useActions } from kitchen sink` but now that I think about it, this is nicer because it is more clear

Reply:
yeah you can still add kitchen-sink as a dependency but because it's all peer dependencies, selectively install dependencies as you expand your functionality, which is nice!